cmd/spiffe-aws-assume-role/cli/cli.go

```diff
@@ -10,7 +10,7 @@ import (
 
 type CLI struct {
 	Credentials              CredentialsCmd   `kong:"cmd,help:'print credentials in a format usable as an AWS credentials_process'"`
-	RolesAnywhereCredentials RolesAnywhereCmd `kong:"cmd,help:'print credentials in a format usable as an AWS credentials_process, generated by RolesAnywhere'"`
+	RolesanywhereCredentials RolesAnywhereCmd `kong:"cmd,help:'print credentials in a format usable as an AWS credentials_process, generated by RolesAnywhere'"`
```

Comment:
This was done specifically because when Kong interpreted RolesAnywhereCredentials it reads as `roles-anywhere-credentials` which is not in line with the actual service name of RolesAnywhere